zone/entity.cpp

```diff
@@ -2334,7 +2350,10 @@ void EntityList::QueueClientsGuild(const EQApplicationPacket *app, uint32 guild_
 
 void EntityList::QueueClientsGuildBankItemUpdate(GuildBankItemUpdate_Struct *gbius, uint32 guild_id)
 {
-	auto outapp = std::make_unique<EQApplicationPacket>(OP_GuildBank, sizeof(GuildBankItemUpdate_Struct));
+	auto outapp = std::make_unique<EQApplicationPacket>(
```

Comment:
This is what I've started to do with packet objects to reduce memory thrash.

```
static EQApplicationPacket p(OP_ClientUpdate, sizeof(PlayerPositionUpdateServer_Struct));	
```
This nails up the memory once and re-uses it for each invocation, this is fine when the process is single threaded

Reply:
Updated.